dev/grunt_options/glotpress_download.js

```diff
@@ -0,0 +1,15 @@
+/**
+ *
+ * Module: glotpress
+ */
+
+module.exports = {
+  core: {
+    options: {
+      url: 'http://translations.tri.be',
+      domainPath: 'lang',
+      slug: 'the-events-calendar',
+      textdomain: 'tribe-events-calendar',
+    },
+  },
```

Comment:
Aren't trailing commas fine in the context of grunt since we are guaranteed to have modern ecmascript?